sda-commons-server-kms-aws/src/test/java/org/sdase/commons/server/kms/aws/config/KmsAwsConfigurationTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sdase.commons.server.kms.aws.config;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class KmsAwsConfigurationTest {
+  KmsAwsConfiguration classUnderTest = new KmsAwsConfiguration();
```

Comment:
Not sure I miss here something, but is `classUnderTest` a shared state between the tests?